src/js/components/help/historyPage.jsx

```diff
@@ -20,21 +20,60 @@ export default class HelpPage extends React.Component {
 
         this.state = {
             history: '',
-			sections: []
+            title: '',
+            clSections: [],
+            tSections: [],
+            mounted: false
```

Comment:
This doesn't have to be in the state. It'd be better to just make it a class variable of `isMounted` like we do for other files. Then you wouldn't have to set state while it's unmounting and create potential issues. It also won't interfere with other things and possibly take longer than it needs to to catch that it unmounted and it'll stay consistent with the rest of the components.

Search for "this.isMounted" for examples of how it's used. GenerateDetachedFilesPage.jsx is a very good example of it